[GIGA Display Shield] Added rotation instructions for SqaureLine (#2255)
arduino/docs-content: content/hardware/10.mega/shields/giga-display-shield/tutorials/10.square-line-tutorial/content.md

}
```

## Rotating the SquareLine UI

> If there is a need to rotate the UI created in SqaureLine on the GIGA Display Shield, there is one line that needs to be added to one of the exported library files. In the **ui -> src** folder, find the `ui.c` file and open it.

Reviewer: ```suggestion
If there is a need to rotate the UI created in SquareLine on the GIGA Display Shield, there is a single line that needs to be added to one of the exported library files. In the `/ui/src` folder, find the `ui.c` file and open it.
```

> The lvgl code we need to add is:

Reviewer: ```suggestion
The code we need to add is:
```

```arduino
lv_display_set_rotation(dispp, rotation);
```

> Where `rotation` can be set to `0`, `90`, `180` or `270` depending on how much you want to rotate the UI.

Reviewer: ```suggestion
The `rotation` can be set to `0`, `90`, `180` or `270` depending on how much you want to rotate the UI.
```
